cerberus_anymal_b_control_1/src/cerberus_anymal_b_control_1/CerberusAnymalBControl1.cpp

```diff
@@ -0,0 +1,44 @@
+/*!
+ * @name      CerberusAnymalBControl1.cpp
+ * @author    Samuel Zimmermann
+ * @email     zsamuel@ethz.ch
+ * @date      May 1, 2020
+ *
+ * Copyright (C) 2020 Robotic Systems Lab, ETH Zurich.
+ * All rights reserved.
+ * http://www.rsl.ethz.ch/
+ */
+
+// ROS
+#include <ros/package.h>
+
+// cerberus_anymal_b_control_1
+#include "cerberus_anymal_b_control_1/CerberusAnymalBControl1.hpp"
+
+namespace cerberus_anymal_b_control_1 {
+
+CerberusAnymalBControl1::CerberusAnymalBControl1(ros::NodeHandle &nodeHandle) :
+  RosController(nodeHandle,12)
```

Comment:
consider to have a const variable with proper name, to make clear it's the number of drives